src/test/unit/types/SdrTest.cpp

```diff
@@ -619,6 +619,29 @@ TEST(SdrTest, TestIntersection) {
     ASSERT_EQ( X.getSum(), 0u );
 }
 
+TEST(SdrTest, TestConcatenationExampleUsage) {
+    SDR A({ 10 });
+    SDR B({ 10 });
+    SDR C({ 20 });
+    A.setSparse(SDR_sparse_t{ 0, 1, 2 });
+    B.setSparse(SDR_sparse_t{ 0, 1, 2 });
+    C.concatenate( A, B );
+    ASSERT_EQ(C.getSparse(), SDR_sparse_t({0, 1, 2, 10, 11, 12}));
+}
+
+TEST(SdrTest, TestConcatenation) {
```

Comment:
axis > 0 is never tested to work, please add a test/example

Reply:
The python test `SDR_ConcatenationTest.testVersusNumpy` tests with axis > 0.